chia/types/blockchain_format/coin.py

```diff
@@ -19,9 +19,16 @@ class Coin(Streamable):
     puzzle_hash: bytes32
     amount: uint64
 
+    def get_hash(self) -> bytes32:
+        # This does not use streamable format for serialization.
```

Comment:
this sentence is a bit misleading too. It *is* using the standard serialization format, just not for hashing purposes